deploy/cperf_ci.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Copyright (c) 2015 Intracom S.A. Telecom Solutions. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v1.0 which accompanies this distribution,
+# and is available at http://www.eclipse.org/legal/epl-v10.html
+
+# Usage: ./cperf_ci [BUILD_NUMBER]
+# Necessary tools installed: docker
+#                            docker-compose
+#                            chmod 777 /opt
+
+BUILD_NUMBER=$1
```

Comment:
Removed. It was introduced as a parameter to avoid overwriting results from a previous run.